File: datasets/IR-Plag/convertai-original-java-rest-python/case-03/plagiarized/L4/04/Kasus3.py
import math

def hitung_tinggi(feet, inches):
    return feet * 12 + inches

def hitung_bmi(weight, feet, inches):
    return weight * 0.45359237 / ((hitung_tinggi(feet, inches) * 0.0254) * (hitung_tinggi(feet, inches) * 0.0254))

if __name__ == "__main__":
    weight = float(input("Enter weight in pounds: "))
    feet = float(input("Enter feet: "))
    inches = float(input("Enter inches: "))
    bmi = hitung_bmi(weight, feet, inches)
    print(f"BMI is {bmi:.2f}")
    if bmi < 18.5:
        print("Underweight")
    elif bmi < 25:
        print("Normal")
    elif bmi < 30:
        print("Overweight")
    else:
        print("Obese")

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-03/plagiarized/L2/04/Kasus3.ts
import * as readline from 'readline';

function hitungTinggi(feet: number, inches: number): number {
    return feet * 12 + inches;
}

function hitungBmi(weight: number, height: number): number {
    return weight * 0.45359237 / ((height * 0.0254) * (height * 0.0254));
}

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question('Enter weight in pounds: ', (weightStr) => {
    const weight = parseFloat(weightStr);
    rl.question('Enter feet: ', (feetStr) => {
        const feet = parseFloat(feetStr);
        rl.question('Enter inches: ', (inchesStr) => {
            const inches = parseFloat(inchesStr);
            const height = hitungTinggi(feet, inches);
            const bmi = hitungBmi(weight, height);
            console.log(`BMI is ${bmi}`);
            if (bmi < 18.5) {
                console.log('Underweight');
            } else if (bmi < 25) {
                console.log('Normal');
            } else if (bmi < 30) {
                console.log('Overweight');
            } else {
                console.log('Obese');
            }
            rl.close();
        });
    });
});

